lighthouse-core/report/html/renderer/text-encoding.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @license Copyright 2021 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
```

Comment:
speaking of https://github.com/GoogleChrome/lighthouse/pull/12570#discussion_r640224422, there used to be a thing with the closure license checker and the files in `renderer/`, which is why they all have the [usual long(er) form of the Apache 2 header](https://github.com/GoogleChrome/lighthouse/blob/2d3059bcd676ddf27fccea89425532bde6b5d203/lighthouse-core/report/html/renderer/category-renderer.js#L1-L16). Have you tried importing this yet? Or is that no longer flagged

Reply:
Seems like something that would only show up in the CL check, which I never did (just built and manually tested). If that's the case, I'll edit in the import and just make a new PR here that can land whenever.

Reply:
> Seems like something that would only show up in the CL check, which I never did (just built and manually tested). If that's the case, I'll edit in the import and just make a new PR here that can land whenever.

Is there any downside to changing it now?

Reply:
snippet renderer has the same comment and is in BUILD, so it should be fine